extensions/binding/org.eclipse.smarthome.binding.astro/README.md

```diff
@@ -10,17 +10,19 @@ This binding supports two Things: Sun and Moon
 
 ## Discovery
 
-Discovery is not necessary, because all calculations are done within the binding.
+You can use the auto discovery if you have a location set in "Configuration - System - Regional Settings - Location". This will automatically create a "Local Sun" and a "Local Moon" item for this location.
+
+If you change your system location the background discovery detects this change and applies it to the "Local Sun" and a "Local Moon" items automatically.
 
 ## Binding Configuration
 
 No binding configuration required.
 
 ## Thing Configuration
 
-A thing requires the geolocation (latitude, longitude) for which the calculation is done.
+A thing requires the geolocation (latitude, longitude, (altitude)) for which the calculation is done.
```

Comment:
The comma should be optional as well. And as commented on the PR, I would prefer the syntax
```
(latitude,longitude[,altitude])
```
especially as we now have two levels of brackets.